packages/ketcher-core/src/domain/entities/struct.ts

```diff
@@ -255,7 +255,8 @@ export class Struct {
     const bidMap = bidMapEntity ?? new Map();
 
     bondSet = bondSet.filter((bid) => {
-      const bond = this.bonds.get(bid)!;
+      const bond = this.bonds.get(bid);
+      if (!bond) return false;
```

Comment:
Which case does it cover here? I suppose we should always have the queried bonds in current struct, if they're not there, perhaps something went wrong and at least we shouldn't just silently return